homework/Truong_Duong/lesson_3/215_Kth_Largest_Element.js

```diff
@@ -0,0 +1,42 @@
+// Solution for https://leetcode.com/problems/kth-largest-element-in-an-array/
+
+/**
+ * @param {number[]} nums
+ * @param {number} k
+ * @return {number}
+ */
+
+function swap(arr, i, j){
+  [arr[i], arr[j]] = [arr[j], arr[i]];
+}
+
+function partition(nums, left, right) {
+  boundary = left;
+  pivot = nums[right];
```

Comment:
I recommend you using `let` or `const` to declare variables.
This code might throw errors when you run javascript with strict mode.

Reply:
Thank you for comment. 
I am now coding with both `python` and `javascript`, that's why sometime I use the wrong syntax 👍

Reply:
Yes, I see, i recommend you using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) and [ESlint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) extension on VSCode, they can throw these issues in writing code context.